RawBencher/RawBencher.csproj

```diff
@@ -76,6 +79,12 @@
     <Reference Include="Microsoft.CodeAnalysis.CSharp, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.CodeAnalysis.CSharp.2.4.0\lib\netstandard1.3\Microsoft.CodeAnalysis.CSharp.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.CodeAnalysis.CSharp.Scripting, Version=2.4.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">

```

Comment:
These are required for Chain?

Reply:
The "compiled" version of Chain uses CS-Script for runtime code generation. It's the one that's pulling in those odd-ball libraries. https://www.nuget.org/packages/CS-Script.bin/

Reply:
That's why "Tortuga.Chain.CompiledMaterializers" is it's own, optional library instead of just being baked into the main package.

Reply:
Ok :)